### DISCUSSION TITLE
docker-compose upgrade to 2.2.5, minio can't startup with old data

### DISCUSSION THREAD

**Discussion by sd1620597** [milvus-io/milvus#23171] [Q&A and General discussion]
`ERROR Unable to use the drive /minio_data: Drive /minio_data: found backend type fs, expected xl or xl-single - to migrate to a supported backend visit https://min.io/docs/minio/linux/operations/install-deploy-manage/migrate-fs-gateway.html: Invalid arguments specified`

backend type is "fs" is not supportted.

minio/.minio.sys/format.json
`{"version":"1","format":"fs","id":"a1f3f977-8b46-48f1-80b5-3f6ca4ef5c63","fs":{"version":"2"}}`

**Comment by yhmo** [comment]
It is minio upgrade issue: https://min.io/docs/minio/linux/operations/install-deploy-manage/migrate-fs-gateway.html
